test-vectors/did_dht/create.json

```diff
@@ -0,0 +1,150 @@
+{
+  "description": "did:dht resolution",
+  "vectors": [
+    {
+      "description": "resolves the vector in https://did-dht.com/#vector-1",
```

Comment:
Shouldn't the vector description be about creation and not resolution?

Reply:
Good catch - I've updated this in [b119689](https://github.com/TBD54566975/web5-kt/pull/181/commits/b11968922fa62e9a9ac3aee4ea0c723f5005a0f1)